persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/PatientMyBatisRepositoryTest.java

```diff
@@ -93,8 +93,8 @@ public void getAllPatientsInStudyDetailedProjection() throws Exception {
             " Invasive Carcinoma project. 825 cases.<br><i>Nature 2012.</i> <a href=\\\"http://tcga-data.nci." +
             "nih.gov/tcga/\\\">Raw data via the TCGA Data Portal</a>.", cancerStudy.getDescription());
         Assert.assertEquals(true, cancerStudy.getPublicStudy());
-        Assert.assertEquals("23000897,26451490", cancerStudy.getPmid());
```

Comment:
multiple PMIDs are supported now.. I think these are test cases added specifically for that. Maybe this PR is out of date with the latest `rc` branch?

Reply:
was the latest when this PR tested. will try again.